### PR TITLE
[docs] Update doc for latest open webui: 0.4.8

### PR DESCRIPTION
**Pull request by Mingqi2**
## Description

Small changes to update webui quickstart doc to support latest 0.4.8 version.

### CODE REVIEW

**Comment by Mingqi2** [comment]
@sgwhat could you please help to review this pr? thanks!

**Comment by Mingqi2** [comment]
@Oscilloscope98 Hi yuwen, as you're the code author, could you please review this PR for doc update? thanks.

File: docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md [hunk 1]
```diff
@@ -163,10 +163,16 @@ Go to **Settings -> Models** in the menu, choose a model under **Pull a model fr
   <img src="https://llm-assets.readthedocs.io/en/latest/_images/open_webui_pull_models.png" width="100%" />
 </a>
 
+> [!NOTE]
+> On latest UI (v0.4.8), the relative settings have been moved to **Admin Settings -> Connections -> Manage Ollama API Connections -> Manage**.
```
Comment (sgwhat) [inline docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md:167]: Change to `On the latest version of Open WebUI`.

File: docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md [hunk 2]
```diff
@@ -163,10 +163,16 @@ Go to **Settings -> Models** in the menu, choose a model under **Pull a model fr
...
+
 
 #### Chat with the Model
 
-Start new conversations with **New chat** in the left-side menu. 
+Start new conversations with **New chat** in the left-side menu.
+> [!RECOMMEND]
+> Disable the auto-completion feature by Open WebUI for good chat experience. Goto **Admin Settings -> Interface -> Autocomplete Generation** and close it.
```
Comment (sgwhat) [inline docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md:174]: Change to `You may disable the auto-completion feature by Open WebUI for an improved chat experience.`

File: docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md [hunk 3]
```diff
@@ -163,10 +163,16 @@ Go to **Settings -> Models** in the menu, choose a model under **Pull a model fr
...
 
+> [!NOTE]
+> On latest UI (v0.4.8), the relative settings have been moved to **Admin Settings -> Connections -> Manage Ollama API Connections -> Manage**.
+
 
 #### Chat with the Model
 
-Start new conversations with **New chat** in the left-side menu. 
+Start new conversations with **New chat** in the left-side menu.
+> [!RECOMMEND]
```
Comment (sgwhat) [inline docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md:173]: This is not a valid Markdown format. Please change it to `[!Note]`.

File: docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md (hunk 2 above)
Comment (sgwhat) [reply docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md:174]: Change `close it` to `disable it`.

File: docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md (hunk 1 above)
Comment (Mingqi2) [reply docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md]: done.

File: docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md (hunk 2 above)
Comment (Mingqi2) [reply docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md]: done.

File: docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md (hunk 3 above)
Comment (Mingqi2) [reply docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md]: done.

**Review by sgwhat — APPROVED** [review]
LGTM

**Pull request merged**